keda-metrics-adapter/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/kedacore/keda-metrics-adapter:1.2.0
+FROM docker.io/kedacore/keda-metrics-adapter:master
```

Comment:
I don't think we should be using master/a moving target

Reply:
KEDA doesn't have an updated version with the fixes yet?

Reply:
fixed.